.github/workflows/npm.yml

```diff
@@ -56,9 +58,14 @@ jobs:
     - run: yarn install --ignore-scripts --frozen-lockfile
       working-directory: node
 
-    - run: npx prebuildify --napi -t ${{ steps.get-nvm-version.outputs.node-version }} --arch arm64
+    - run: |
+        if [[ "${{ matrix.os }}" == "ubuntu-latest" ]]; then
+          sudo apt-get install -y crossbuild-essential-arm64
+          export CARGO_TARGET_AARCH64_UNKNOWN_LINUX_GNU_LINKER=aarch64-linux-gnu-gcc
+        fi
```

Comment:
Nitpick: I think this would be clearer as its own build step (but the actual command seems fine)

Reply:
Updated - thanks for the feedback!

Here's another example run to confirm that it works: https://github.com/dennisameling/libsignal-client/runs/5239772453?check_suite_focus=true